h2o-core/src/main/java/water/H2O.java

```diff
@@ -1875,6 +1867,38 @@ public static void raw_remove(Key key) {
     if( v != null ) v.removePersist();
   }
   public static void raw_clear() { STORE.clear(); }
+
+  /**
+   * Cleans H2O's DKV while retaining specified collection of {@link Key}s.
+   * Only Frame or Model Keys are retained, others are silently ignored.
```

Comment:
We agreed with @michalkurka only Frames and Models are retainable. Others are silently ignored.